internal/config/config.go

```diff
@@ -629,6 +637,20 @@ func cidrContainsCIDR(outer, inner *net.IPNet) bool {
 	return false
 }
 
+func lowestMask(cidrs []*net.IPNet) int {
+	if len(cidrs) == 0 {
+		return 0
+	}
+	lowest, _ := cidrs[0].Mask.Size()
+	for _, c := range cidrs {
+		s, _ := c.Mask.Size()
+		if lowest > s {
+			lowest, _ = c.Mask.Size()
```

Comment:
Why not `lowest = s`?

Reply:
`internal/config/config.go:648:11: assignment mismatch: 1 variable but c.Mask.Size returns 2 values`

Reply:
Sorry, now I get it. You are right.

Reply:
Fixed